sites/public/src/components/listings/ListingsMap.tsx

```diff
@@ -45,7 +46,13 @@ const ListingsMap = (props: ListingsMapProps) => {
   const [infoWindowIndex, setInfoWindowIndex] = useState(null)
 
   return isLoaded ? (
-    <GoogleMap mapContainerStyle={containerStyle} center={center} zoom={9}>
+    <GoogleMap
```

Comment:
so like (pesudocode alert)
```html
<div class="listings-map">
  <DoorwayMapZoom...
  <GoogleMap..
</div>

Reply:
Can you elaborate please?

Reply:
This is an extension of the previous comment to move the component into this level

Reply:
Im am showing some pseudocode to demonstrate how to do this